kokoro/gcp_windows/upload_symbols.ps1

```diff
@@ -19,23 +19,26 @@ function get_authorization_header {
 
 function get_crash_symbol_collector_key_name($authorization) {
   $keys = Invoke-RestMethod -Header $authorization  -ContentType "application/json" -Uri "https://apikeys.googleapis.com/v2beta1/projects/60941241589/keys"
-  if (-Not $keys) { Throw "Can't get list of API keys" }
+  if (-Not $keys.keys) {
+    Write-Host $keys
```

Comment:
Can you explain this? I guess this makes the key appear in the build log? Is that a problem? Do we have to consider the key to be a secret?

Reply:
Yes, sorry for not doing it right away. My logic is 

- If we get a right response, we will always have this structure:
{
  "keys": [
    {
      key1 info
    },
    {
      key2 info
    }
  ]
}
and in this case (-Not $keys.keys) is false and we won't print the keys in the log.

- But if the request returns a error code the $keys will be empty or 
{
  "error": {
     the error info
  }
}
In both cases we won't see the real key, but it would be useful for us to know the reason. 

(The same for Linux.)

Reply:
Sounds good. Thank you for the explanation.